osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -256,8 +256,29 @@ public void PostCommand(string text, Channel target = null)
                     JoinChannel(channel);
                     break;
 
+                case "chat":
+                case "msg":
+                case "query":
+                    if (string.IsNullOrWhiteSpace(content))
+                    {
+                        target.AddNewMessages(new ErrorMessage($"Usage: /{command} [user]"));
+                        break;
+                    }
+
+                    // Check if the user has joined requested channel already.
+                    var alreadyJoinedChannel = JoinedChannels.FirstOrDefault(
+                        c => c.Type == ChannelType.PM && c.Users.Count == 1 && c.Name.Equals(content, StringComparison.OrdinalIgnoreCase));
+                    if (alreadyJoinedChannel != null)
+                        CurrentChannel.Value = alreadyJoinedChannel;
```

Comment:
Should be unnecessary to query user ID if already found channel no?

```suggestion

                    if (alreadyJoinedChannel != null)
                    {
                        CurrentChannel.Value = alreadyJoinedChannel;
                        break;
                    }
```